### DISCUSSION TITLE
Plot stuck at last stage for hours

### DISCUSSION THREAD

**Discussion by joelrb** [Chia-Network/chia-blockchain#5041] [Support]
Hi,

I've been waiting for hours for plotting to complete moving the temp file to my final plot directory. Seems like nothing is going on anymore (no write activity in the ssd).

In temp dir I have this:
plot-k32-2021-05-13-09-15-abd22d9e44358d5539b971190d170328420c40a77c1355a0fb4f0e1e3fa3f516.plot.2.tmp
106,231,738 KB

In plot dir I have this:
plot-k32-2021-05-13-09-15-abd22d9e44358d5539b971190d170328420c40a77c1355a0fb4f0e1e3fa3f516.plot.2.tmp    106,231,738 KB

This is like 6 hours past 100% in GUI log in Windows.

Can I safely rename this to .plot?

Thanks!

**Comment by cccat6** [comment]
What system you are using? Check the drive monitor and see if it writing to the final dir. If so, just wait. If not, I guess you can try rename.

**Reply by joelrb** [reply]
Windows 10. Yes it's written to final dir with same filename and bytes as temp dir. It's more than 10hrs already since 100%.

Here's screenshot of file in final dir. It shows modified date. I wonder if it's still writing?

![Screenshot 2021-05-15 130715](https://user-images.githubusercontent.com/225284/118348689-a42f4480-b57e-11eb-9779-251c9752509b.png)

**Reply by cccat6** [reply]
No, that didn't show anything. See "Resource Monitor" on your computer. Look close to these parts. If high active time shows, that is it's working.
![image](https://user-images.githubusercontent.com/22387156/118348901-dd1be900-b57f-11eb-93a9-0650f54be283.png)

**Reply by joelrb** [reply]
![Screenshot 2021-05-15 132306](https://user-images.githubusercontent.com/225284/118349034-bb6f3180-b580-11eb-8d50-21654ec9cb7f.png)

Hmm I wonder what caused the slowdown. It was doing 10-12hrs max in my previous plots. I restarted the computer before I created this new plot. Other than Chia, only edge browser is active.

**Reply by cccat6** [reply]
I am sure your plot is writing. It seems really slow. About 6MB/s (from your screenshot). This means you will need to wait about 5 hours for a single plot to finish coping.
You might need to wait longer.

**Reply by joelrb** [reply]
Yeah it's weird for this plot. It completed the 10hrs mark and then stuck writing to final dir for more than 10hrs now. It was ok in previous plots. I guess I'll just have to wait. Thanks.

**Reply by cccat6** [reply]
A friendly reminder. Disable your Antimalware Service. That maybe will get some improvement. I did that on my machine and it works. Windows default will check any file write and run. That takes a ton of system resources.

**Reply by joelrb** [reply]
> A friendly reminder. Disable your Antimalware Service. That maybe will get some improvement. I did that on my machine and it works. Windows default will check any file write and run. That takes a ton of system resources.

Ok thanks for the reminder.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
